src/sinks/aws_cloudwatch_logs/config.rs

```diff
@@ -65,6 +67,9 @@ pub struct CloudwatchLogsSinkConfig {
     /// For example, you might choose `host`.
```

Comment:
nit: seems like we shouldn't be using the word "you" directly in the documentation.

Also, since one of the examples is "{{ host }}" , this seems unnecessary.

Reply:
Resolved in [972d219](https://github.com/vectordotdev/vector/pull/16408/commits/972d2194532039169cd79f698de089681170a98a), and found another use of "you/r"